.config/dotnet-tools.json

```diff
@@ -0,0 +1,24 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "fake-cli": {
+      "version": "5.20.3",
+      "commands": [
+        "fake"
+      ]
+    },
+    "paket": {
+      "version": "6.0.0-beta4",
+      "commands": [
+        "paket"
+      ]
+    },
+    "fable": {
+      "version": "3.0.0-nagareyama-rc-011",
+      "commands": [
+        "fable"
+      ]
```

Comment:
Considering that not so long ago each of these tools came from a different place and had different ways to manage versions, it's so nice to see them together now :)